src/adservice/src/main/java/hipstershop/AdService.java

```diff
@@ -221,101 +197,29 @@ private static void initStats() {
       logger.info("Stats disabled.");
       return;
     }
-    logger.info("Stats enabled");
+    logger.info("Stats enabled, but temporarily unavailable");
```

Comment:
Should we maybe link to the GitHub issue for these messages, e.g.:
"Stats enabled, but temporarily unavailable. See <URL> for details"

Reply:
Done.